web/content/js/nav.js

```diff
@@ -12,7 +12,7 @@ function buildNavigation() {
 
   return `<nav>
   <a href="/">
-    <img class="logo" src="/img/helmet.svg" alt="Home">
+    <img class="logo" src="/img/helmet.png" alt="Home">
```

Comment:
What's the reason to replace the SVG with a PNG?

Reply:
The reason is that I got carried away. I'm about to break this commit and a couple of more into a separate PR and will make a note in there to address this.

Reply:
Fair enough. Happens to me all the time :)